ui/src/app/shared/components/input-filter.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {Autocomplete} from '../../../../node_modules/argo-ui';
+import {ErrorPanel} from './error-panel';
+
+interface InputProps {
+    value: string;
+    placeholder: string;
+    type: string;
+    onChange: (input: string) => void;
+}
+
+interface InputState {
+    input: string;
```

Comment:
is this the current value? if so maybe call it value?

Reply:
Changed to `value`.